src/diagnose.ts

```diff
@@ -189,6 +195,10 @@ export class Diagnostics {
 
 
 		function showErrorAt(start: number, end: number, errorMessage: string, severity: vscode.DiagnosticSeverity, code: DiagnosticCode) {
+			if (suppressedWarningsArray.includes(code)) {
```

Comment:
Wouldn't it be a nicer API to write the code names themselves, and not an integer, into the user configuration?

Reply:
Why not! Do you mean something like this?

```jsonc
"arbEditor.suppressedWarnings": [
    "mismatched_brackets",
    "metadata_for_missing_key",
    "invalid_key",
    "missing_metadata_for_key",
    "invalid_placeholder",
    "missing_other_in_icu",
    "unknown_icu_message_type",
    "placeholder_without_metadata",
    "missing_placeholder_with_metadata",
    "missing_messages_from_template",
] // or "all"
```

Reply:
Yes exactly. this should then just be documented somewhere, in the readme for example.